### PR TITLE
Disable login and logout for local development (#399).

### PR DESCRIPTION
**Pull request by inferno-chromium**

### CODE REVIEW

**Comment by inferno-chromium** [comment]
Basically disables login, logout and switch account since firebase api key does not work locally ?

File: src/appengine/handlers/corpora.py [hunk 1]
```diff
@@ -25,9 +25,9 @@
 class Handler(base_handler.Handler):
   """Manage data bundles."""
 
+  @handler.unsupported_on_local_server
```
Comment (oliverchang) [inline src/appengine/handlers/corpora.py]: any reason for the order changes in this PR?

Comment (inferno-chromium) [reply src/appengine/handlers/corpora.py]: It only mattered for

```
class LogoutHandler(base_handler.Handler):
  """Log out handler."""	  """Log out handler."""


   @handler.unsupported_on_local_server
  @handler.require_csrf_token	  @handler.require_csrf_token
  @handler.get(handler.HTML)	  @handler.get(handler.HTML)
```
where wrong order would mean we show the csrf error first. so i thought to fix the others as well. it is just unnecessary work, when instead we should be bailing out early.

**Pull request merged**